troubleshooting/index.md

```diff
@@ -39,10 +39,10 @@ If you’re seeing this message, the page you’d like to use GhostText on doesn
 
 If you’re seeing this message, the browser extension can’t find the GhostText server running in an editor. Ensure that:
 
-- Your editor is open
-- Its GhostText extension is installed; Install it from the [list of supported editors](/#installation) if you haven’t already.
-- The GhostText server is running; In most editor extensions this is started automatically.
-- The server port matches; It’s 4001 by default, it can be changed in the options.
-- There are no other servers using the port. For example what happens when you open http://localhost:4001? The page should return details about WebSocket and the editor should start a new connection.
+- Your editor is launched
+- GhostText editor extension is installed; install it from the [list of supported editors](/#installation) if you haven’t already.
```

Comment:
I prefer the previous one, because like this it sounds like there's *one* "GhostText editor extension"

Also US style guides they capitalize the letter after the colons and semicolon when there's a full sentence, so that should be undone everywhere https://www.grammarly.com/blog/capitalization-after-colons/

Reply:
> I prefer the previous one, because like this it sounds like there's *one* "GhostText editor extension"

I forgot why I made that change and now it doesn't look right even to myself 😅

> Also US style guides they capitalize the letter after the colons and semicolon when there's a full sentence, so that should be undone everywhere https://www.grammarly.com/blog/capitalization-after-colons/

TIL 👍 Will do